src/main/java/ca/corefacility/bioinformatics/irida/database/changesets/AutomatedAnalysisToTemplate.java

```diff
@@ -0,0 +1,168 @@
+	private IridaWorkflowsService workflowsService;
+	private DataSource dataSource;
+
+	@Override
+	public SqlStatement[] generateStatements(Database database) throws CustomChangeException {
+
+		JdbcTemplate jdbcTemplate = new JdbcTemplate(dataSource);
+
+		IridaWorkflow assemblyWorkflow = null;
+		IridaWorkflow sistrWorkflow = null;
+
+		//get the workflow information from the service
+		try {
+			assemblyWorkflow = workflowsService.getDefaultWorkflowByType(BuiltInAnalysisTypes.ASSEMBLY_ANNOTATION);
+		} catch (IridaWorkflowNotFoundException e) {
+			logger.warn(
+					"Assembly workflow not found.  Automated assemblies will not be converted to analysis templates.");
+			//Note this will definitely happen in the galaxy CI tests as only SNVPhyl and a test workflow are configured.
+		}
+
+		//get the workflow information from the service
+		try {
+			sistrWorkflow = workflowsService.getDefaultWorkflowByType(BuiltInAnalysisTypes.SISTR_TYPING);
+		} catch (IridaWorkflowNotFoundException e) {
+			logger.warn("SISTR workflow not found.  Automated SISTR will not be converted to analysis templates.");
+			//Note this will definitely happen in the galaxy CI tests as only SNVPhyl and a test workflow are configured.
+		}
+
+		//update assemblies
+		if (assemblyWorkflow != null) {
+			logger.debug("Upadting automated assembly project settings");
+
+			//get the workflow identifiers
+			UUID assemblyId = assemblyWorkflow.getWorkflowIdentifier();
+			//get the default parameters
+			List<IridaWorkflowParameter> defaultAssemblyParams = assemblyWorkflow.getWorkflowDescription()
+					.getParameters();
+
+			//insert the assembly
+			insertWorkflow(jdbcTemplate, "Automated AssemblyAnnotation", true, assemblyId, "p.assemble_uploads=1",
+					defaultAssemblyParams);
+
+		}
+
+		//update SISTR
+		if (sistrWorkflow != null) {
+			logger.debug("Upadting automated SISTR project settings");
+			//get the workflow identifiers
+			UUID sistrId = sistrWorkflow.getWorkflowIdentifier();
+
+			//get the default params
+			List<IridaWorkflowParameter> defaultSistrParams = sistrWorkflow.getWorkflowDescription()
+					.getParameters();
+
+			//insert the sistr entries without metadata
+			insertWorkflow(jdbcTemplate, "Automated SISTR Typing", false, sistrId, "p.sistr_typing_uploads='AUTO'",
+					defaultSistrParams);
+			//insert the sistr entries with metadata
+			insertWorkflow(jdbcTemplate, "Automated SISTR Typing", true, sistrId,
+					"p.sistr_typing_uploads='AUTO_METADATA'", defaultSistrParams);
+		}
+
+		return new SqlStatement[0];
+	}
+
+	private void insertWorkflow(JdbcTemplate jdbcTemplate, String name, boolean updateSamples, UUID workflowId,
+			String where, List<IridaWorkflowParameter> params) {
+		/*
+		 * we're borrowing the 'automated' flag here to mark entries we need to add params to later.  at this point
+		 * there'll be nothing with a '1' in the automated flag.  we'll clear it later.
+		 */
+
+		int updateSampleBit = updateSamples ? 1 : 0;
```

Comment:
Would a `boolean` work here, since that's what the `updateSampleBit` is representing? Not too sure about if it needs to be `0` or `1` or `false` or `true` here.

Reply:
This was done just since a `bit(1)` type is a `1` or `0`, but yeah mysql is smart enough to translate a `true` or `false` in the query.  Removed this in 0b67979ce1